README.md

```diff
@@ -145,25 +146,7 @@ Links to wiki, forum and issues.
 After successful configuration, go to Settings -> Devices & Services and add the integration 'Smart Irrigation'
 You should end up with one device and one entity for each zone and their attributes, listed below as well as [seven services](#available-services).
 
-#### Services
-For each instance of the component the following services will be available:
-
-| Service | Description|
-| --- | --- |
-|`Smart Irrigation: calculate_zone`|Triggers the calculation of one specific zone.|
-|`Smart Irrigation: calculate_all_zones`|Triggers the calculation of all automatic zones. Use only if you disabled automatic refresh in the options.|
-|`Smart Irrigation: update_zone`|Updates one specific zone with weather data|
-|`Smart Irrigation: update_all_zones`|Updates all automatic zones with weather data|
-|`Smart Irrigation: reset_bucket`|Resets one specific bucket.|
-|`Smart Irrigation: reset_all_buckets`|Resets all automatic buckets.|
-|`Smart Irrigation: set_bucket`|Sets a specific bucket to to a specific `value`.|
-|`Smart Irrigation: set_all_buckets`|Sets all automatic buckets to a specific `value`.|
-
-#### Start Event
-
-| Event | Description|
-| --- | --- |
-|`[instance]_start`|Fires depending on `sensor.[zone_name]` value and sunrise. You can listen to this event to optimize the moment you irrigate so your irrigation starts just before sunrise and is completed at sunrise. See below for examples on how to use this.|
+For each instance of the component the following entities, services and events will be available:
```

Comment:
there is only one instance, so this first part can be updated.

Reply:
instance, object, class,entity, integration ... sometime I will learn it. :-D

For the the component the following entities, services and events will be available:

Ok?

Reply:
How about: "Once the component is installed, the following entities, services and events will be available:"?

Reply:
Nice